addons/zeus/stringtable.xml

```diff
@@ -1186,6 +1197,7 @@
         </Key>
         <Key ID="STR_ACE_Zeus_ModuleGarrison_DisplayName">
             <English>Garrison Group</English>
+            <German>Garnisonsgruppe</German>
```

Comment:
Afaik Garrison module means "besetzen" as in "Gebäude besetzen"
should check if vanilla also has a translation for garrison and see what they do

Reply:
https://synixebrett.github.io/stringtables/ mixed results

Reply:
Aber das Garrison modul ist ja dazu da um gebäude zu besetzen.
Nicht um Garnisionen zu gruppieren.

Reply:
Hab leider wenig Ahnung von Zeus, daher hoff ich auf Vorschläge :D

Reply:
Yes Garrison means "Gebäude besetzen" not "Garnision"
No idea how to name the module though..
Maybe just "Gebäude besetzen"
Also I think the other translations also thought it meant "Garnision" and translated it as that